workflow/controller/workflowpod.go

```diff
@@ -460,7 +460,9 @@ func (woc *wfOperationCtx) createWorkflowPod(ctx context.Context, nodeName strin
 		}
 		created, err := woc.controller.kubeclientset.CoreV1().ConfigMaps(woc.wf.ObjectMeta.Namespace).Create(ctx, cm, metav1.CreateOptions{})
 		if err != nil {
-			return nil, err
+			if !apierr.IsAlreadyExists(err) {
+				return nil, err
+			}
 		}
 		woc.log.Infof("Created configmap: %s", created.Name)
```

Comment:
This log message is now misleading, could you fix it (I would suggest two possible logs, one for created and one for already exists)..

Reply:
fixed, thanks.

Reply:
ok

Reply:
fixed